hibernate-core/src/test/java/org/hibernate/orm/test/cfg/persister/GoofyPersisterClassProvider.java

```diff
@@ -775,6 +775,16 @@ public void forEachAttributeMapping(Consumer<? super AttributeMapping> action) {
 		public JavaType getMappedJavaType() {
 			return null;
 		}
+
+		@Override
+		public org.hibernate.persister.entity.Loadable asLoadable() {
+			return ( org.hibernate.persister.entity.Loadable ) this;
```

Comment:
the cast should be unnecessary? That's the point of using this pattern.

Reply:
Also this method is unnecessary if you added the `asLoadable` method on the `Loadable` interface.